src/main/java/projectbookstore/dto/BookDto.java

```diff
@@ -0,0 +1,17 @@
+package projectbookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
We can use "Data" in our dto's, because they are not working with DB, but it's still okay.